### ISSUE TITLE
[Question] Does it support hot reload?

### ISSUE THREAD

**Issue by ali50m** [natemcmaster/dotnet-serve#148]
I would like to serve some html/css/js things, does this tool support hot reload? Thanks!

**Comment by natemcmaster** [comment]
No, it doesn't support this. I'm not sure how to implement it either. If you have ideas and want to contribute, feel free to propose something.
